src/main/asciidoc/jdbc.adoc

```diff
@@ -1,73 +1,71 @@
-Among others it tracks changes to entities.
+Among other things, it tracks changes to entities.
 It does lazy loading for you.
-It allows to map a wide array of object constructs to an equally wide array of database design.
+It lets you map a wide array of object constructs to an equally wide array of database designs.
 
 This is great and makes a lot of things really easy.
 Just take a look at a basic JPA tutorial.
-But often it gets really confusing why JPA does a certain thing.
-Or things that are really simple conceptually get rather difficult with JPA.
+But it often gets really confusing as to why JPA does a certain thing.
+Also, things that are really simple conceptually get rather difficult with JPA.
 
-Spring Data JDBC aims to be much simpler conceptually:
+Spring Data JDBC aims to be much simpler conceptually, by supporting the following features:
 
-* If you load an entity, SQL statements get executed and once this is done you have a completely loaded entity.
+* If you load an entity, SQL statements get executed.
+Once this is done, you have a completely loaded entity.
 No lazy loading or caching is done.
 
-* If you save and entity it gets saved.
-If you don't it doesn't.
+* If you save an entity, it gets saved.
+If you do not, it does not.
 There is no dirty tracking and no session.
 
-* There is a simplistic model of how to map entities to tables.
+* There is a simple model of how to map entities to tables.
 It probably only works for rather simple cases.
-If you don't like that, just code your strategy yourself.
-Spring Data JDBC will offer only very limited support for customizing the strategy via annotations.
+If you do not like that, you should code your own strategy.
+Spring Data JDBC offers only very limited support for customizing the strategy with annotations.
 
 [[jdbc.domain-driven-design]]
 === Domain Driven Design and Relational Databases.
 
-All Spring Data modules are inspired by the concepts of Repository, Aggregate and Aggregate Root from Domain Driven Design.
-These are possibly even more important for Spring Data JDBC because they are to some extend contrary to normal practice when working with relational databases.
+All Spring Data modules are inspired by the concepts of "`repository`", "`aggregate`", and "`aggregate`" root from Domain Driven Design.
+These are possibly even more important for Spring Data JDBC, because they are, to some extent, contrary to normal practice when working with relational databases.
 
-An *Aggregate* is a group of entities that is guaranteed to be consistent between atomic changes to it.
+An aggregate is a group of entities that is guaranteed to be consistent between atomic changes to it.
 A classic example is an `Order` with `OrderItems`.
-A property on `Order`, e.g. `numberOfItems` will be consistent with the actual number of `OrderItems`.
+A property on `Order` (for example, `numberOfItems`) is consistent with the actual number of `OrderItems`.
```

Comment:
the example does not end with `numberOfItems` but goes all the way to the end of the sentence.

Reply:
That makes a sentence fragment, so I had to extend the sentence there.